website/docs/r/cdn_frontdoor_rule_set.html.markdown

```diff
@@ -32,8 +32,6 @@ resource "azurerm_cdn_frontdoor_rule_set" "example" {
 
 ## Arguments Reference
 
-The following arguments are supported:
```

Comment:
why's this been removed/can we add this back in?

Reply:
Fixed.